lib/base/logger.cpp

```diff
@@ -242,6 +254,12 @@ Log::~Log()
 		 * then cout will not flush lines automatically. */
 		std::cout << std::flush;
 	}
+
+#ifdef _WIN32
+	if (Logger::IsEarlyLoggingEnabled() && entry.Severity >= Logger::GetConsoleLogSeverity()) {
```

Comment:
This `&&` should be `||` shouldn't it?

Reply:
No, that would spam the WEL (Windows Event Log) with debug messages during startup. After early logging gets disabled, messages should only be logged to the WEL when it is explicitly configured (this will then happen in the loop above).